blocks/user-licenses/user-licenses.js

```diff
@@ -1,27 +1,421 @@
-import { getEnvironmentConfig, getLocaleConfig } from '../../scripts/zemax-config.js';
-import { createTag } from '../../scripts/scripts.js';
+import { getLocaleConfig } from '../../scripts/zemax-config.js';
+import {
+  createTag, createGenericTable, hideModal, showModal,
+} from '../../scripts/scripts.js';
+import execute from '../../scripts/zemax-api.js';
+
+function showUserActionModal(event) {
+  const newProductUserId = event.target.getAttribute('data-new-productuserid');
+  const licenseId = event.target.getAttribute('data-license-id');
+  const nextActionClass = event.target.getAttribute('data-next-action-class');
+  showModal(event);
+  const userActionButton = document.querySelector(`.${nextActionClass}`);
+  userActionButton.setAttribute('data-new-productuserid', newProductUserId);
+  userActionButton.setAttribute('data-license-id', licenseId);
+}
+
+function createUser(event) {
+  hideModal(event);
+  alert('Implement create user');
```

Comment:
Will there be a follow-up issue to remove this?

Reply:
Yes, https://github.com/hlxsites/zemax/issues/164
there is lot of functionality and whole new page needs to be developed for managing and creating the user
![image](https://github.com/hlxsites/zemax/assets/129520313/b02881ed-c396-426b-8c6d-6fe323a12fd5)
You can see the functionality by login to zemax.com 
username: Alice.Example@ansys.com
password: TestExampleContact!